roles/percona-server/tasks/main.yml

```diff
@@ -46,6 +46,29 @@
   until: result|succeeded
   retries: 5
 
+# Upgrade to the latest xtradb for CVEs
+# This needs to be done serially because it'll restart the damn thing on
+# package upgrade.
+- block:
+  - name: upgrade percona to desired version
+    apt:
+      pkg: percona-xtradb-cluster-server-{{ xtradb.server_version }}
+      state: latest
```

Comment:
So the server_version is a simple string, like `5.6`. Because the package _name_ is actually `percona-xtradb-cluster-server-5.6` which then has a complete version string _after_ that for the package version. It's confusing, I know, but our role default is mostly about the major version (5.6) rather than the minor, which is covered with `latest`.